tutorials/nextjs-app-serverless-functions-sqldb/index.mdx

```diff
@@ -0,0 +1,948 @@
+          "PGPORT" = trimprefix(regex(":[0-9]{1,5}",scaleway_sdb_sql_database.database.endpoint), ":")
+      }
+      secret_environment_variables = {
+        "PGPASSWORD" = scaleway_iam_api_key.api_key.secret_key,
+      }
+    }
+      
+    resource scaleway_iam_application "app" {
+    name = "tutorial-nextjs-app-tf"
+    }
+      
+    resource scaleway_iam_policy "db_access" {
+    name = "tutorial-nextjs-policy-tf"
+    description = "Gives tutorial Nextjs app access to Serverless SQL Database"
+    application_id = scaleway_iam_application.app.id
+    rule {
+      project_ids = ["${var.DEFAULT_PROJECT_ID}"]
+      permission_set_names = ["ServerlessSQLDatabaseReadWrite"]
+    }
+    }
+      
+    resource scaleway_iam_api_key "api_key" {
+    application_id = scaleway_iam_application.app.id
+    }
+      
+    resource scaleway_sdb_sql_database "database" {
+    name = "tutorial-nextjs-tf"
+    min_cpu = 0
+    max_cpu = 8
+    }
+      
+    output "database_connection_string" {
+    // Output as an example, you can give this string to your application
+    value = format("postgres://%s:%s@%s",
+      scaleway_iam_application.app.id,
+      scaleway_iam_api_key.api_key.secret_key,
+      trimprefix(scaleway_sdb_sql_database.database.endpoint, "postgres://"),
+    )
+    sensitive = true
+    }
+      
+    output "container_url" {
+    // Output as an example, you can give this string to your application
+    value = scaleway_container.main.domain_name
+    sensitive = true
+    }
+    ```
+
+The terraform file creates several resources:
+
+- A [Serverless Containers namespace](https://www.scaleway.com/en/docs/serverless/containers/concepts/#namespace), that contains a [Serverless Container](https://www.scaleway.com/en/docs/serverless/containers/quickstart/), which hosts your Next.js application
+
+- A [Serverless SQL Database](https://www.scaleway.com/en/docs/serverless/sql-databases/reference-content/serverless-sql-databases-overview/) which stores posts data
+
+- An [IAM policy](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#policy) which grants your application the right permissions
+
+- An [IAM application](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#application), used as a principal for your IAM policy
+
+- An [IAM secret key](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#api-key) used as credentials to authenticate your application to the database
+
+### Deploying the app with Terraform
+
+1. Run the following command to initialize the Terraform working directory:
+    ```bash
+    terraform init
+    ```
+
+2. Add the `REGISTRY_ENDPOINT` and `DEFAULT_PROJECT_ID` environment variables to Terraform:
+    ```bash
+    export TF_VAR_REGISTRY_ENDPOINT=$REGISTRY_ENDPOINT
+    export TF_VAR_DEFAULT_PROJECT_ID=$(scw config get default-project-id) 
+    ```
+
+3. Create and review Terraform the execution plan of your infrastructure:
```

Comment:
Feels like there is something missing here, between 'and review' and 'Terraform'.